app/models/job_io_wrapper.rb

```diff
@@ -0,0 +1,79 @@
+# Primarily for jobs like IngestJob to revivify an equivalent FileActor to one that existed on
+# the caller's side of an asynchronous Job invocation.  This involves providing slots
+# for the metadata that might travel w/ the actor's various supported types of @file.
+# For example, we cannot just do:
+#
+#   SomeJob.perform_later(arg1, arg2, File.new('/path/to/file'))
+#
+# Because we'll get:
+#
+#   ActiveJob::SerializationError: Unsupported argument type: File
+#
+# This also applies to Hydra::Derivatives::IoDecorator, Tempfile, etc., pretty much any IO.
```

Comment:
How could this apply to a Tempfile?  That would be removed as soon as the process ended, right?